Pipfile

```diff
@@ -11,13 +11,18 @@ pytest = "*"
 black = "*"
 pytest-cov = "*"
 mypy = "*"
+sqlalchemy-stubs = "*"
 
 [packages]
 flask = "*"
 psycopg2 = "*"
```

Comment:
Yes, because flask_alchemy does not have stubs for mypy, so it has to be a combination of sqlalchemy_stubs and mypy config to be able to run it without warnings.

Reply:
Sorry I was referring to psycopg2.